src/lib/containers/drug_upload_tool/WarningModal.tsx

```diff
@@ -0,0 +1,36 @@
+import React, { useState } from 'react';
+import Button from 'react-bootstrap/Button'
+import Modal from 'react-bootstrap/Modal'
+
+export type WarningModalProps = {
+  title: string
+  copy: string | JSX.Element
+  confirmCopy?: string
+  show: boolean
+  onOK: Function
+  onCancel: Function
+  callbackArgs: any
+}
+
+export default function WarningModal(props: WarningModalProps) {
+  const [show] = useState(props.show);
```

Comment:
it doesn't seem like `show` is being set anywhere, so the modal component could set `show` from props directly ```JSX <Modal show={props.show}.../>```and then in UserFileGrid on line 264 you can set ```JSX <WarningModal show={this.state.modalContext}.../>``` and not do conditional rendering since `show` already defines whether WarningModal gets rendered or not

Reply:
good point. was left over from old implementation. will remove